### PR TITLE
Updating the SDL2 release package links

### PR DESCRIPTION
**Pull request by bseto**
Ever since I moved to Ubuntu 17.10, the old packages were giving me lots of "SDL redeclared" errors. 

Updating the SDL2 packages for me fixed it. 

@enochtsang or @ctlai95, please test to see if this works on your mac/16.04LTS

### CODE REVIEW

**Comment by ExtraGravity** [comment]
Travis is having bad times, gotta fix that first.

**Comment by ctlai95** [comment]
I made some changes, mind if I push to this branch?

**Comment by ExtraGravity** [comment]
@ctlai95 push away

File: installer.sh [hunk 1]
```diff
@@ -2,18 +2,17 @@
...
 unameOut="$(uname -s)"
 case "${unameOut}" in
-    Linux*)     MACHINE=Linux;;
-    Darwin*)    MACHINE=Mac;;
-    CYGWIN*)    MACHINE=Cygwin;;
-    MINGW*)     MACHINE=MinGw;;
+    Linux*)     MACHINE=Linux ;;
+    Darwin*)    MACHINE=Mac ;;
+    CYGWIN*)    MACHINE=Cygwin ;;
+    MINGW*)     MACHINE=MinGw ;;
```
Comment (ExtraGravity) [inline installer.sh:8]: why

File: installer.sh [hunk 2]
```diff
@@ -104,8 +107,8 @@ cd $DIR
 # glog
 git clone https://github.com/google/glog.git
 cd glog
-mkdir build
-cd build
+mkdir tmp
+cd tmp
```
Comment (ExtraGravity) [inline installer.sh:115]: what's wrong with build

File: installer.sh [hunk 3]
```diff
@@ -89,13 +90,15 @@ cd $DIR
...
 ./configure
 make -j4
 make install -j4
-cd ../../
+cd $DIR
+
+#SMPEG
+wget https://www.libsdl.org/projects/smpeg/release/smpeg2-2.0.0.tar.gz
+tar xzf smpeg2-2.0.0.tar.gz
+cd smpeg2-2.0.0
```
Comment (ExtraGravity) [inline installer.sh:104]: why did this change?

File: installer.sh (hunk 1 above)
Comment (ctlai95) [reply installer.sh:8]: Auto change from editor

File: installer.sh (hunk 2 above)
Comment (ctlai95) [reply installer.sh:115]: On my mac it complains that build already exists

File: installer.sh (hunk 3 above)
Comment (ctlai95) [reply installer.sh:104]: For me smpeg didn't exist in the previous section

File: installer.sh (hunk 2 above)
Comment (ExtraGravity) [reply installer.sh:115]: instead of creating /tmp, maybe we can just cd in to build without making it.

**Review by ExtraGravity — CHANGES_REQUESTED** [review]

File: installer.sh (hunk 2 above)
Comment (ctlai95) [reply installer.sh:115]: TIL apple's HFS is case-insensitive by default, and that's why it's conflicting with the file `BUILD` in `glog/`

what do

File: installer.sh (hunk 3 above)
Comment (bseto) [reply installer.sh:104]: Actually, same with me. My installer always stopped at 
```
./installer.sh: line 94: cd: external/smpeg2-2.0.0: No such file or directory
```

My usual fix was to just ignore it and comment it out, and re-run the script.

File: installer.sh (hunk 2 above)
Comment (bseto) [reply installer.sh:115]: We could just do `mkdir -p build`.
Adding the -p should stop it from complaining if the `build/` directory already exists

Comment (ctlai95) [reply installer.sh:115]: I still get `mkdir: build: Not a directory`

**Review by ExtraGravity — APPROVED** [review]

**Pull request closed without merging**